tests/test_rfc3739.py

```diff
@@ -111,6 +112,19 @@ class SequenceOfOID(univ.SequenceOf):
         self.assertEqual(2, count)
         self.assertTrue(found_qc_stmt_oid)
 
+class WithComponentsTestCase(unittest.TestCase):
+
+    def testDerCodec(self):
+
+        try:
+            si = rfc3739.SemanticsInformation()
+            substrate = der_encoder(si)
+            encode_without_error = True
+        except error.PyAsn1Error:
```

Comment:
There is an idiom for this in unittest:

    self.assertRaises(error.PyAsn1Error, der_encode, si)

effectively replacing the whole try/except block.

Does it make sense?

Reply:
I did not know about assertRaises().  Nice improvement!

Reply:
I just added the improved test_rfc3739.py to the PR